emscons.py

```diff
@@ -17,5 +17,6 @@
 
 env = os.environ.copy()
 env['EMSCRIPTEN_TOOL_PATH'] = tool_path
+env['EMSCRIPTEN_ROOT'] = utils.path_from_root()
```

Comment:
I wonder if we can just have the scons tools find the emcc tools relative to itself? 

Is `tools/scons/site_scons/site_tools/emscripten` ever install outside of the emscripten root?

Reply:
I thought we were getting rid of `EMSCRIPTEN_ROOT`? 

Can we submit the change to get rid of `EMSCRIPTEN_ROOT` as its own standalone change?

Reply:
Apparently the idea was to make `tools/scons/site_scons/site_tools/emscripten/emscripten.py` not _require_ `EMSCRIPTEN_ROOT`, and assume that the tools will be in the `PATH` as a fallback. But `emscons` could still provide it, why not?

Or do you mean "to get rid of" in a more project-wide sense?

Sorry I don't remember much of this anymore lol. Am just trusting past me.

Reply:
I was hoping that we can completely get rid of it.   

I guess that only works if we can assume that `tools/scons/site_scons/site_tools/emscripten/emscripten.py` itself always lives in the emscripten directory.. which maybe we can't?    What do you think?

Reply:
Sounds reasonable to me.
So we look for the binaries based on `dirname(__file__)`? or `EMSCRIPTEN_TOOL_PATH`? (with the proper `..`s of course)

Reply:
Actually maybe we can leave this as is... to keep the file portable.